kura/org.eclipse.kura.web2/META-INF/MANIFEST.MF

```diff
@@ -69,7 +69,7 @@ Import-Package: com.eclipsesource.json;version="0.9.5",
  org.eclipse.kura.net.status.ethernet;version="[1.0,2.0)",
  org.eclipse.kura.net.status.modem;version="[1.0,2.0)",
  org.eclipse.kura.net.status.wifi;version="[1.0,2.0)",
- org.eclipse.kura.net.wifi;version="[2.2,3.0)",
+ org.eclipse.kura.net.wifi;version="[2.4,2.5)",
```

Comment:
We might want to discuss this choice with the rest of team. We can keep it this way if we plan to maintain `web2` in the distribution forever.